www/src/components/ColorModePicker.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+const storageKey = 'recharts-color-mode';
+
+type ColorModeStrategy = 'light' | 'dark' | 'system';
+
+function getStoredColorModeStrategy(): ColorModeStrategy {
+  const rawStoredValue = localStorage.getItem(storageKey);
```

Comment:
Do we have try/catch around localStorage? Some environments will throw.

Reply:
addressed in the new defineColoreModeStore module